data_pipeline/surveymonkey/surveymonkey_etl.py

```diff
@@ -0,0 +1,30 @@
+import http.client
+import json
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+SURVEYMONKEY_ACCESS_TOKEN_ENV_VAR = "SURVEYMONKEY_ACCESS_TOKEN"
+
+
+def get_connection_surveymonkey_api() -> http:
+    return http.client.HTTPSConnection("api.surveymonkey.com")
+
+
+def get_surveymonkey_api_headers() -> dict:
+    return {
+        'Accept': "application/json",
+        'Authorization': f"Bearer {SURVEYMONKEY_ACCESS_TOKEN_ENV_VAR}"
+        }
+
+
+def get_survey_list() -> list:
+    conn = get_connection_surveymonkey_api()
+    headers = get_surveymonkey_api_headers()
+
+    conn.request("GET", "/v3/surveys", headers=headers)
```

Comment:
Just a thought:

Using the `http` library seems quite manual. I am not even sure whether that is actually using HTTPS?
I find the `requests` library much easier to use. e.g.:

```python
import requests

response = requests.get('https://api.surveymonkey.com/v3/surveys', headers=headers)
response.raise_for_status()
return response.json()['data']
```